docs/feature_flags.md

```diff
@@ -61,3 +61,11 @@ Exemplar storage is implemented as a fixed size circular buffer that stores exem
 This takes the snapshot of the chunks that are in memory along with the series information when shutting down and stores
 it on disk. This will reduce the startup time since the memory state can be restored with this snapshot and m-mapped 
 chunks without the need of WAL replay.
+
+## Extra Scrape Metrics
+
+`--enable-feature=extra-scrape-metrics`
+
+This adds additional automatically generated metrics for each scrape target.
```

Comment:
```suggestion
This inserts additional metrics in TSDB for each scrape target.
```

Let's make it clear that it is in TSDB and not in the /metrics endpoint.

Reply:
I was following the [language in the docs](https://prometheus.io/docs/concepts/jobs_instances/#automatically-generated-labels-and-time-series).

Reply:
The language is: `For each instance scrape, Prometheus stores a sample in the following time series:`

`stores a sample` makes it clear that it is TSDB.

`add a metric` is less clear to me.


What about:

```suggestion
When enabled, for each instance scrape, Prometheus stores a sample in the following additional time series:
```

Reply:
Sure, Done.